src/client/components/organisms/paging-lookup/paging-lookup.jsx

```diff
@@ -145,32 +145,37 @@ class PagingLookup extends React.Component {
   }
 
   createSbaOfficeFilterList(sbaOffices) {
-    if (sbaOffices === null || sbaOffices.length === 0) {
-      return sbaOffices
-    }
+    if (Array.isArray(sbaOffices) === true && sbaOffices.length > 0) {
```

Comment:
```suggestion
    if (Array.isArray(sbaOffices) && sbaOffices.length > 0) {
```

I think this will work